packages/libs/react-ui/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  checkpointContainerStyle,
+  circleColorVariant,
+  circleStyle,
+  lineColorVariant,
+  lineContainerStyle,
+  lineStyle,
+  progressBarStyle,
+  textContainerStyle,
+} from './ProgressBar.css';
+
+import classNames from 'classnames';
+import React, { FC } from 'react';
+
+export interface IProgressBarProps {
+  checkpoints: ICheckpoint[];
+}
+
+export interface ICheckpoint {
+  title: string;
+  status: 'complete' | 'pending' | 'incomplete';
+}
+
+export const ProgressBar: FC<IProgressBarProps> = ({ checkpoints }) => {
+  return (
+    <div
+      className={progressBarStyle}
+      data-testid="kda-progress-bar"
+      key={'progress-bar'}
```

Comment:
since it's not in a loop doesn't need a key

Reply:
Key's can be used outside of loops as well btw, see; https://www.youtube.com/watch?v=vXJkeZf-4-4 or read; https://kentcdodds.com/blog/understanding-reacts-key-prop 😁